site/content/en/guides/consumer/get/_index.md

```diff
@@ -174,6 +208,16 @@ Use `kubectl apply` to deploy the local package to a remote cluster.
 
 ### Apply Command
 
+{{% hide %}}
+
+<!-- @createKindCluster @verifyGuides-->
+```
+kind delete cluster && kind create cluster
```

Comment:
Should we also add a step to install kind it it's not present?

Reply:
I don't think we should do this in the guide itself. Ideally we would like to avoid spinning up kind for every guide, but that risks situations where we have dependencies between the tests so I decided to avoid that now. 
This assumes that both mdrip and kind is available to run the tests. Unlike what we do in cli-utils, in this setup we don't try to install anything in the makefile, but just installs this in the github action before we start the tests.